diff --git a/node_modules/glob/dist/commonjs/glob.js b/node_modules/glob/dist/commonjs/glob.js
index 278ef30..f586640 100644
--- a/node_modules/glob/dist/commonjs/glob.js
+++ b/node_modules/glob/dist/commonjs/glob.js
@@ -77,7 +77,7 @@ class Glob {
         if (!opts.cwd) {
             this.cwd = '';
         }
-        else if (opts.cwd instanceof URL || opts.cwd.startsWith('file://')) {
+        else if (opts.cwd instanceof node_url_1.URL || opts.cwd.startsWith('file://')) {
             opts.cwd = (0, node_url_1.fileURLToPath)(opts.cwd);
         }
         this.cwd = opts.cwd || '';
